### PR TITLE
[ADD] mrp_account_analytic_wip

### PR DESCRIPTION
**Pull request by dreispt**
Depends on:
- [x] mrp_analytic_cost #607 (bundled!)
- [ ] account_analytic_wip https://github.com/OCA/account-analytic/pull/360

Supersedes #599 and #605:
The previous `mrp_account_material_wip` and `mrp_account_work_order_wip` modules are merged into a single one `mrp_account_analytic_wip`.

### CODE REVIEW

**Comment by dreispt** [comment]
@marcelsavegnago @CasVissers-360ERP Setting this as ready to review.

**Comment by dreispt** [comment]
image

image

**Comment by dreispt** [comment]
@Chandresh-OSI please review

File: mrp_analytic_cost/views/mrp_workcenter_view.xml
Comment (patrickrwilson) [inline mrp_analytic_cost/views/mrp_workcenter_view.xml]: This line is no longer needed (since v11)

File: mrp_account_analytic_wip/views/mrp_production_views.xml
Comment (patrickrwilson) [inline mrp_account_analytic_wip/views/mrp_production_views.xml]: This line no longer needed (since v11)

**Review by patrickrwilson — APPROVED** [review]
Very small change, no blockers. LGTM if Travis and Build are green after the dependencies are merged.

**Comment by dreispt** [comment]
Superseded by #657

**Pull request closed without merging**
